lib/openstax/rescue_from/configuration.rb

```diff
@@ -2,10 +2,54 @@ module OpenStax
   module RescueFrom
     class Configuration
 
-      # attr_accessor, attr_reader, attr_writers for variables
+      attr_accessor :raise_exceptions, :system_logger, :notifier,
+        :html_template_path, :layout_name, :application_name, :exception_status_codes,
+        :non_notifying_exceptions, :exception_extras,:friendly_status_messages
 
       def initialize
-        # set default values for variables
+        @raise_exceptions = false
+        @application_name = 'Tutor'
+        @system_logger = Rails.logger
+        @notifier = ExceptionNotifier
+        @html_template_path = 'errors/any'
```

Comment:
Makes sense. This is also how I did it in Commontator (see https://github.com/lml/commontator/blob/master/lib/tasks/commontator_tasks.rake for the copy task)